### PR TITLE
[Withings] Fix some problems with the README.

### LINKED ISSUE #5281: [Withings] 'consumerKey' configuration key is unknown
Binding throws the following exeption:
```
2017-08-21 00:08:45.350 [ERROR] [org.apache.felix.configadmin        ] - [org.osgi.service.cm.ManagedService, id=1171, bundle=217/file:/usr/share/openhab2/addons/org.openhab.binding.withings-1.10.0.jar]: Updating property consumerKey of configuration org.openhab.withings-oauth caused a problem: The given configuration key is unknown!
```
## Expected Behavior
After trowing "org.openhab.binding.withings-1.10.0.jar" to the addons-folder this binding should setup itself.

## Possible Solution / Context
* I am not sure about the reasons. I tried to restart the binding.
Found that withings is now NOKIA. But URLs used in binding (WithingsAuthenticator.java) are still alive.
Minimum there should be an Update of the URLs
* I also tried to use the steps here [NOKIA-Health-API](https://developer.health.nokia.com/api) with DEFAULT_CONSUMER_KEY and DEFAULT_CONSUMER_SECRET given in WithingsAuthenticator.java.
Was able to find my withings-data when followed every step.
* Tried also to build my own **withings.cfg** like that (used ***** for privacy reasons):
```
token=04d08e***********************39fdcd06b4fe054c04
tokensecret=02c3fb2*********************de33c1cb1bc836bf774994c1
``` 
That gives the same error-messages
* Using the WIKI-Page was not very helpful because there are some contradictory informations. eg.:
`To change the OAuth parameters for the Withings binding the user can specify the following values in the services/withings.cfg:
The OAuth configuration must be stored in the file services/withings-oauth.cfg`

## Environment
* Version used: org.openhab.binding.withings-1.10.0.jar
* Operating System: openHABian (latest snapshot 2.2.0~20170820133825-1 (Build #1020)

## [DEBUG] log of start / stop
```
2017-08-21 00:08:43.581 [DEBUG] [org.openhab.binding.withings        ] - BundleEvent STARTING - org.openhab.binding.withings
2017-08-21 00:08:43.812 [DEBUG] [org.openhab.binding.withings        ] - BundleEvent STARTED - org.openhab.binding.withings
2017-08-21 00:08:44.376 [DEBUG] [org.openhab.binding.withings        ] - ServiceEvent REGISTERED - {org.osgi.service.cm.ManagedService}={service.pid=org.openhab.withings-oauth, osgi.command.function=[startAuthentication,finishAuthentication], component.name=org.openhab.binding.withings.authenticator, component.id=309, osgi.command.scope=withings, service.id=1171, service.bundleid=217, service.scope=bundle} - org.openhab.binding.withings
2017-08-21 00:08:45.350 [ERROR] [org.apache.felix.configadmin        ] - [org.osgi.service.cm.ManagedService, id=1171, bundle=217/file:/usr/share/openhab2/addons/org.openhab.binding.withings-1.10.0.jar]: Updating property consumerKey of configuration org.openhab.withings-oauth caused a problem: The given configuration key is unknown!
org.osgi.service.cm.ConfigurationException: consumerKey : The given configuration key is unknown!
	at org.openhab.binding.withings.internal.api.WithingsAuthenticator.updated(WithingsAuthenticator.java:251) [217:org.openhab.binding.withings:1.10.0]
	at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updated(ManagedServiceTracker.java:189) [7:org.apache.felix.configadmin:1.8.14]
	at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updateService(ManagedServiceTracker.java:152) [7:org.apache.felix.configadmin:1.8.14]
	at org.apache.felix.cm.impl.helper.ManagedServiceTracker.provideConfiguration(ManagedServiceTracker.java:85) [7:org.apache.felix.configadmin:1.8.14]
	at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.provide(ConfigurationManager.java:1463) [7:org.apache.felix.configadmin:1.8.14]
	at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.run(ConfigurationManager.java:1419) [7:org.apache.felix.configadmin:1.8.14]
	at org.apache.felix.cm.impl.UpdateThread.run0(UpdateThread.java:141) [7:org.apache.felix.configadmin:1.8.14]
	at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:109) [7:org.apache.felix.configadmin:1.8.14]
	at java.lang.Thread.run(Thread.java:745) [?:?]
2017-08-21 00:08:45.444 [DEBUG] [org.openhab.binding.withings        ] - ServiceEvent REGISTERED - {org.openhab.model.item.binding.BindingConfigReader, org.openhab.binding.withings.WithingsBindingProvider}={component.name=org.openhab.binding.withings.genericbindingprovider, component.id=311, service.id=1172, service.bundleid=217, service.scope=bundle} - org.openhab.binding.withings
2017-08-21 00:08:47.763 [DEBUG] [org.openhab.binding.withings        ] - ServiceEvent REGISTERED - {org.osgi.service.cm.ManagedService}={service.pid=org.openhab.withings, component.name=org.openhab.binding.withings.binding, component.id=310, service.id=1174, service.bundleid=217, service.scope=bundle} - org.openhab.binding.withings
```

### PR DESCRIPTION
**Pull request by 9037568**
It appears a number of updates to the binding code were never documented properly. This should improve the README.

Fixes #5281.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.withings/README.md [hunk 1]
```diff
@@ -1,89 +1,123 @@
...
 | Diastolic Blood Pressure | diastolic_blood_pressure | mmHg |
 | Systolic Blood Pressure | systolic_blood_pressure | mmHg |
-| Heart Pulse | heart_pulse | bpm |
+| Heart Pulse     | heart_pulse | bpm |
 
 
 ## Examples
 
-The following snippet shows some sample bindings:
+### Examples of multiple account configuration
```
Comment (mboremski) [inline bundles/binding/org.openhab.binding.withings/README.md]: Could we add a single-user example here too, pls?

**Comment by mboremski** [comment]
Could you also add a logging info (e.g. level DEBUG) here:
[Line 138..141 of WithingsAccount.java](https://github.com/9037568/openhab/blob/425a7dac3b1f23a1be5846d97ecafd3591fd7962/bundles/binding/org.openhab.binding.withings/src/main/java/org/openhab/binding/withings/internal/api/WithingsAccount.java#L138)

File: bundles/binding/org.openhab.binding.withings/README.md (hunk 1 above)
Comment (9037568) [reply bundles/binding/org.openhab.binding.withings/README.md]: My understanding is no special setup is needed for a single user.

**Comment by 9037568** [comment]
This is a situation that probably should never occur, but sure, I'll add a
couple of DEBUGs in there.

**Pull request merged**